tools/event-subscriber/Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o event-subscriber
 FROM scratch
 LABEL source=git@github.com:kyma-project/kyma.git
 
+WORKDIR /app
```

Comment:
?? why do we change the workdir to /app and not change it in the `CMD` in line 27? imho this introduces an error